dali/test/dali_plugin_manager_test.cc

```diff
@@ -27,34 +27,47 @@ static const std::string& DummyPluginLibPath() {
 namespace other_ns {
 
 class DummyTest : public ::dali::DALITest {
-};
+ public:
+  static void LoadDummyPlugin() {
+    ::dali::PluginManager::LoadLibrary(DummyPluginLibPath());
+  }
 
-static void LoadDummyPlugin() {
-  ::dali::PluginManager::LoadLibrary(DummyPluginLibPath());
-}
+  void TestPlugin(const std::string &backend) {
+    LoadDummyPlugin();
+
+    dali::TensorList<dali::CPUBackend> data;
+    dali::test::MakeRandomBatch(data, 3);
 
-TEST_F(DummyTest, PluginShouldBeUsable) {
-  LoadDummyPlugin();
+    std::array<std::string, 2> backends = {"cpu", "gpu"};
+    for (auto backend : backends) {
```

Comment:
This for loop does the same thing as the `this->TestPlugin("cpu"/"gpu");`.
You can remove it.